test_googleMapsApiModule.py

```diff
@@ -4,8 +4,9 @@
 
```

Comment:
Not completely convinced if the filename should be test_googleMapsApiModule.py. Let us add this file to /tests/googleMapsApiModule.py. What do you think @athityakumar

Reply:
@vishakha-lall `pytest` requires the prefix or suffix of testing file to be `test_*.py` or `*_test.py` for test discovery.

Reply:
I see, okay then